src/explorer/astro.config.mjs

```diff
@@ -70,7 +70,6 @@ export default defineConfig({
     react(),
     partytown(),
   ],
-  experimental: {
```

Comment:
Is prettier not running on .mjs files or something?

Reply:
I was using my dotnet profile and ran format after i remembered.